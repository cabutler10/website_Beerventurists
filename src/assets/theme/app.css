html {
  height: 100%;
}

body {
  margin: 0;
  padding: 0;
  font-family: 'Open Sans', sans-serif;
  background-color: #ffffff;
  overflow-x: hidden;
  font-size: 14px;
  width: '100%';
}

a {
  text-decoration: none;
}
