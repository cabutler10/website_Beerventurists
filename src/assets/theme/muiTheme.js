import { createMuiTheme } from '@material-ui/core/styles'
import 'typeface-open-sans'
import 'typeface-permanent-marker'
import './app.css'

const status = {
  white: '#ffffff',
  black: '#040001',
  greyLt: '#F7F5F5',
  greyMed: '#656565',
  grey: '#c5c6c7',
  greyDk: '#404040',
  green: '#00942b',
  yellow: '#fce566',
  red: '#FF0012',
  blue: '#015b99',
  blueLt: '#0172c0',
  blueDk: '#014472',
  orange: '#c56113',
  orangeDk: '#c56113',
  orangeLt: '#ebab2f',
  tan: '#FFFBEF',
}

const theme = createMuiTheme({
  palette: {
    type: 'light',
    primary: {
      light: status.blueLt,
      main: status.blue,
      dark: status.blueDk,
    },
    secondary: {
      light: status.orangeLt,
      main: status.orange,
      dark: status.orangeDk,
    },
    error: { main: status.red },
  },
  status,
  typography: {
    useNextVariants: true,
    fontFamily: "'Open Sans', sans-serif",
    h5: {
      letterSpacing: 3,
    },
    h2: {
      fontFamily: "'Permanent Marker', cursive",
    },
    body1: {
      letterSpacing: 1,
      lineHeight: 1.8,
      //TODO is this an error in MUI? not sure why its default is 16 when MUI default is 14
      fontSize: 14,
    },
    body2: {
      letterSpacing: 1,
      fontSize: 14,
    },
    button: {
      letterSpacing: 3,
    },
  },
  overrides: {
    MuiTypography: {
      colorPrimary: {
        color: status.black,
      },
      colorSecondary: {
        color: status.orangeLt,
      },
      colorTextSecondary: {
        color: status.white,
      },
    },
    MuiButton: {
      root: {
        padding: '9px 27px',
        borderRadius: 0,
      },
      contained: {
        boxShadow: 'none',
      },
    },
  },
})

export default theme
