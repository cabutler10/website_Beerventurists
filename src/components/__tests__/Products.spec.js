import React from 'react'
import { render } from 'react-testing-library'
import Products from '../Products'
import { ThemeProvider } from '@material-ui/styles'
import theme from '../../assets/theme/muiTheme'

describe('Products', () => {
  it('renders correctly', () => {
    const mockProps = {
      t: () => '',
    }
    const component = render(
      <ThemeProvider theme={theme}>
        <Products {...mockProps} />
      </ThemeProvider>
    )
    expect(component).toMatchSnapshot()
  })
})
