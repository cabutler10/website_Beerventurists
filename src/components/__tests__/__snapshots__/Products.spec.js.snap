// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Products renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Hook-root-1ntpn8u"
      >
        <h5
          class="MuiTypography-root-1 MuiTypography-h5-17 Hook-title-30w93p"
        >
          products.sectionTitle
        </h5>
        <div
          class="Hook-container-13v418p"
        >
          <div
            class="Hook-itemContainer-vdh5bf"
          >
            <img
              alt="product 0"
              class="Hook-img-cai670"
              src="test-file-stub"
            />
            <div
              class="Hook-descriptionContainer-1tsden"
            >
              <p
                class="MuiTypography-root-1 MuiTypography-body1-10 Hook-name-uohml8"
              >
                FCS Double Tab Thruster
              </p>
              <a
                class="MuiButtonBase-root-63 MuiButton-root-37 MuiButton-outlined-45 MuiButton-outlinedPrimary-46 Hook-button-429z7t"
                href="https://amzn.to/2OfsJjc"
                rel="noopener"
                role="button"
                tabindex="0"
                target="_blank"
              >
                <span
                  class="MuiButton-label-38"
                >
                  common.more
                </span>
                <span
                  class="MuiTouchRipple-root-66"
                />
              </a>
            </div>
          </div>
          <div
            class="Hook-itemContainer-vdh5bf"
          >
            <img
              alt="product 1"
              class="Hook-img-cai670"
              src="test-file-stub"
            />
            <div
              class="Hook-descriptionContainer-1tsden"
            >
              <p
                class="MuiTypography-root-1 MuiTypography-body1-10 Hook-name-uohml8"
              >
                RailProtect Tape
              </p>
              <a
                class="MuiButtonBase-root-63 MuiButton-root-37 MuiButton-outlined-45 MuiButton-outlinedPrimary-46 Hook-button-429z7t"
                href="https://amzn.to/2Y7BDDQ"
                rel="noopener"
                role="button"
                tabindex="0"
                target="_blank"
              >
                <span
                  class="MuiButton-label-38"
                >
                  common.more
                </span>
                <span
                  class="MuiTouchRipple-root-66"
                />
              </a>
            </div>
          </div>
          <div
            class="Hook-itemContainer-vdh5bf"
          >
            <img
              alt="product 2"
              class="Hook-img-cai670"
              src="test-file-stub"
            />
            <div
              class="Hook-descriptionContainer-1tsden"
            >
              <p
                class="MuiTypography-root-1 MuiTypography-body1-10 Hook-name-uohml8"
              >
                Future Single Tab Thruster
              </p>
              <a
                class="MuiButtonBase-root-63 MuiButton-root-37 MuiButton-outlined-45 MuiButton-outlinedPrimary-46 Hook-button-429z7t"
                href="https://amzn.to/2OkhqGA"
                rel="noopener"
                role="button"
                tabindex="0"
                target="_blank"
              >
                <span
                  class="MuiButton-label-38"
                >
                  common.more
                </span>
                <span
                  class="MuiTouchRipple-root-66"
                />
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Hook-root-1ntpn8u"
    >
      <h5
        class="MuiTypography-root-1 MuiTypography-h5-17 Hook-title-30w93p"
      >
        products.sectionTitle
      </h5>
      <div
        class="Hook-container-13v418p"
      >
        <div
          class="Hook-itemContainer-vdh5bf"
        >
          <img
            alt="product 0"
            class="Hook-img-cai670"
            src="test-file-stub"
          />
          <div
            class="Hook-descriptionContainer-1tsden"
          >
            <p
              class="MuiTypography-root-1 MuiTypography-body1-10 Hook-name-uohml8"
            >
              FCS Double Tab Thruster
            </p>
            <a
              class="MuiButtonBase-root-63 MuiButton-root-37 MuiButton-outlined-45 MuiButton-outlinedPrimary-46 Hook-button-429z7t"
              href="https://amzn.to/2OfsJjc"
              rel="noopener"
              role="button"
              tabindex="0"
              target="_blank"
            >
              <span
                class="MuiButton-label-38"
              >
                common.more
              </span>
              <span
                class="MuiTouchRipple-root-66"
              />
            </a>
          </div>
        </div>
        <div
          class="Hook-itemContainer-vdh5bf"
        >
          <img
            alt="product 1"
            class="Hook-img-cai670"
            src="test-file-stub"
          />
          <div
            class="Hook-descriptionContainer-1tsden"
          >
            <p
              class="MuiTypography-root-1 MuiTypography-body1-10 Hook-name-uohml8"
            >
              RailProtect Tape
            </p>
            <a
              class="MuiButtonBase-root-63 MuiButton-root-37 MuiButton-outlined-45 MuiButton-outlinedPrimary-46 Hook-button-429z7t"
              href="https://amzn.to/2Y7BDDQ"
              rel="noopener"
              role="button"
              tabindex="0"
              target="_blank"
            >
              <span
                class="MuiButton-label-38"
              >
                common.more
              </span>
              <span
                class="MuiTouchRipple-root-66"
              />
            </a>
          </div>
        </div>
        <div
          class="Hook-itemContainer-vdh5bf"
        >
          <img
            alt="product 2"
            class="Hook-img-cai670"
            src="test-file-stub"
          />
          <div
            class="Hook-descriptionContainer-1tsden"
          >
            <p
              class="MuiTypography-root-1 MuiTypography-body1-10 Hook-name-uohml8"
            >
              Future Single Tab Thruster
            </p>
            <a
              class="MuiButtonBase-root-63 MuiButton-root-37 MuiButton-outlined-45 MuiButton-outlinedPrimary-46 Hook-button-429z7t"
              href="https://amzn.to/2OkhqGA"
              rel="noopener"
              role="button"
              tabindex="0"
              target="_blank"
            >
              <span
                class="MuiButton-label-38"
              >
                common.more
              </span>
              <span
                class="MuiTouchRipple-root-66"
              />
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
